mypy/test/testmodulefinder.py

```diff
@@ -1,18 +1,62 @@
 import os
+from typing import Iterator, List, Tuple
 
 from mypy.options import Options
 from mypy.modulefinder import (
     FindModuleCache,
     SearchPaths,
     ModuleNotFoundReason,
-    expand_site_packages
 )
 
 from mypy.test.helpers import Suite, assert_equal
 from mypy.test.config import package_path
 data_path = os.path.relpath(os.path.join(package_path, "modulefinder"))
 
 
+def expand_site_packages(site_packages: List[str]) -> Tuple[List[str], List[str]]:
```

Comment:
These functions seem to be in support of something, but I don't see what they are supposed to be used for. Perhaps you missed adding some test code before pushing?

Reply:
They were used in the `ModuleFinderSitePackagesSuite` tests, but the test data doesn't really need to use `.pth` files anymore because we don't need to test `mypy.modulefinder.expand_site_packages()` anymore. So I've removed the `.pth` files from the test data and listed the packages directories manually.